index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <div class="wrapper">
+            <header>
+                <nav class="nav-bar">
+                    <a href="#">
+                        <div class="logo">My Logo</div>
+                        <img src="/images/Enabled.png" alt="enable-image">
+                      </a>
+                </nav>
+            </header>
+            <div class="contents">
+                <section class="main-section">
+                    <h1 class="heading">I’m Derrick
+                        Glad to see you!</h1>
+                        <p class="intro-1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        <h3 class="intro-2">LET'S CONNECT</h3>
+                        <ul class="nav-links">
+                            <li>
+                                <a href="#"><img src="/images/twitter.png" alt="twitter"></a>
+                            </li>
+                            <li>
+                                <a href="#"><img src="/images/Linkedin.png" alt="linkedin"></a>
+                            </li>
+                            <li>
+                                <a href="#"><img src="/images/m.png" alt="m icon"></a>
+                            </li>
+                            <li>
+                                <a href="#"><img src="/images/github.png" alt="github"></a>
+                            </li> <li>
+                                <a href="#"><img class="thumbs-up" src="/images/Vector@2x.png" alt="Vector"></a>
```

Comment:
[OPTIONAL]
Please correct your image path:
instead of '/images/name_image' write 'images/name_image'